### DISCUSSION TITLE
Uninstalling IRAF from a MAC

### DISCUSSION THREAD

**Discussion by jparmas** [iraf-community/iraf#176] [Q&A] [answered]
How can I uninstall IRAF from a MAC and turn back to how it was before I installed it?

**Comment by olebole** [comment, marked as answer]
You should remove the IRAF directory tree and the directory `.iraf` in your home dir. Then remove the following lines from your shell login scripts:

```sh
# Add iraf setup commands
if [ -e /users/jparmas/.iraf/setup.sh ]; then
    source /users/jparmas/.iraf/setup.sh
fi
```
They however don't harm when the `.iraf` directory was removed.

**Comment by jparmas** [comment]
Thank you very much Ole for the response!!
The reason I want to remove it is because I followed the instructions to
install it in my MAC OS and didn't work.
Maybe you can help me with some instructions to make it work in my mac
before I go ahead and erase it. Mine is Macbook Air 8GB 13-inch, Mid 2013.
I would really appreciate it we can make it work in my machine.
I already run the Terminal script that comes with the files in the
[iraf-2.16.1-2021.06.14] version and it seems to be installed, but It wont
open...
THANKSSSS!!!

Juan Pablo

**Reply by olebole** [reply]
Please write exactly what you did. Also, did you run the tests? What was the result? What is the "terminal script"? etc.

**Reply by olebole** [reply]
One general comment: IRAF is really legacy code. I would not recommend to start anything new with IRAF. This will be a dead end. Instead, consider Astropy for analysis, which is already quite complete. Time is better invested in improving astropy than in trying to get IRAF running.

**Comment by jparmas** [comment]
By the way, these lines you suggested to erase from the shell login
scripts in case of removing IRAF, are THE ONLY LINES of my .bash_login file
Is it safe to erase the .bash_login file??
Or I need to erase them somewhere else? Thanks!
Thanks

JPA

**Reply by olebole** [reply]
You can remove it completely.

**Comment by jparmas** [comment]
The truth is that your comment makes a lot of sense to me. I will give it a try and spend my time on astropy and uninstall IRAF. 
Thank you very much for your help!!!
JP
